plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -219,6 +221,21 @@ public BigQueryConfig setCaseInsensitiveNameMatchingCacheTtl(Duration caseInsens
         return this;
     }
 
+    @NotNull
+    @MinDuration("0m")
+    public Duration getDestinationTableCacheExpiry()
+    {
+        return destinationTableCacheExpiry;
+    }
+
+    @Config("bigquery.destination-table-cache-expiry")
```

Comment:
maybe `bigquery.views-cache-ttl` would be a better name

Reply:
Done.